giraffa-core/src/main/java/org/apache/giraffa/hbase/ClientNamenodeProtocolServerSideCallbackTranslatorPB.java

```diff
@@ -259,7 +291,8 @@ public void getAdditionalDatanode(RpcController controller,
     try {
       done.run(blockingTranslator.getAdditionalDatanode(controller, req));
     } catch (ServiceException e) {
-      handleRemoteException(controller, e, done);
+      handleRemoteException(controller, e, done,
+          GiraffaPBHelper.getAdditionalDatanode());
```

Comment:
the same here (and all other handleRemoteException() calls with some empy fake objects)